src/src/pages/capabilities/summary/index.jsx

```diff
@@ -236,18 +236,18 @@ export default function Summary({ anchorId }) {
         <div className={styles.column} style={{ paddingTop: "2rem" }}>
           <MyMembershipApplication />
           <ButtonStack align="right">
-            {canJoin ? (
+            {canJoin && (
```

Comment:
<img width="2968" height="1868" alt="Screenshot 2025-12-02 at 11  04 41@2x" src="https://github.com/user-attachments/assets/195a9879-4a13-4ada-ba22-c1b69621f139" />
I see both join and leave button for a Capability that I'm not a member of.